packages/fiori3/src/index.ts

```diff
@@ -16,6 +16,8 @@ import { CustomListItem } from './lib/CustomListItem';
 import { DatePicker } from './lib/DatePicker';
 import { DeviationIndicator } from './lib/DeviationIndicator';
 import { Dialog } from './lib/Dialog';
+import { FilterBar } from './lib/FilterBar';
```

Comment:
after you have created all required files in the lib folder, please run `node packages/fiori3/scripts/create-library-export.js`